delivery_ups_oca/README.rst

```diff
@@ -38,16 +40,17 @@ Configuration
 To configure this module, you need to:
 
 #. Add options to your UPS carrier
+#. Go to *Inventory > Configuration* and check "Delivery Packages" option.
 #. Add a carrier account with delivery type ``ups`` and fill in your credentials
 
 Usage
 =====
 
-To use this module, you need to:
+You have to set Ups carrier in the stock picking you want to ship:
 
-#. Select the ups carrier on your sale orders or pickings
-#. Set exactly one delivery option
-#. Press the button to generate labels for a done picking
+ * In the stock picking form go to *Additional Info* tab and choose Ups as carrier and the service and product code. You only be able to choose this if the state of the picking is 'Ready to Transfer.
```

Comment:
Don't forget these are user instructions, not developer ones, so use UI terminology:
```suggestion
* In the delivery order form go to *Additional Info* tab and choose the UPS shipping
  method you created as carrier and the service and product code.
* You will be only able to choose this if the state of the delivery is "Ready to Transfer".
```
Another thing is that the choosing should be done also on state "Partially available".

Reply:
Also service code is defined on the carrier and should be in CONFIGURE.rst.